### DISCUSSION TITLE
Is there any support for sleap-anipose (3D)?

### DISCUSSION THREAD

**Discussion by delaroob** [talmolab/sleap#1725] [Help!] [answered]
Hello everyone, 

I'm trying to make 3D analysis work and it seemed logical to use sleap-anipose for triangulating 2D sleap data but I'm having some issues (charuco board corners are detected very innacurately during calibration). 
The github page for the program seems a little dead, so I was just wondering if there is a way to get help with this?

Thanks in advance!

**Comment by ericleonardis** [comment, marked as answer]
Hello! Yes, I just replied to your issue on the sleap-anipose repo. Thanks for bringing this to our attention, let's get to the bottom of this.
